contracts/AKAP.sol

```diff
@@ -24,6 +24,7 @@ contract AKAP is ERC721Full {
         uint seeAlso;
         address seeAddress;
         bytes nodeBody;
+        string tokenURI;
```

Comment:
There is no need to store this on the node because it's already stored in ERC721Metadata. If you remove this and the nodes[..].tokenURI = .. below it should be good I think.